dask/dataframe/shuffle.py

```diff
@@ -778,7 +779,7 @@ def drop_overlap(df, index):
 
 
 def get_overlap(df, index):
-    return df.loc[[index]] if index in df.index else None
+    return df.loc[[index]] if index in df.index else df._constructor()
```

Comment:
maybe instead of relying on `_constructor` we could do `type(df)()` ?

Reply:
Ah, apparently Tom prefers `_constructor ` https://github.com/dask/dask/pull/5137#discussion_r422348273 .  Please disregard.